prow/config/config.go

```diff
@@ -1559,6 +1560,74 @@ func (cfg *SlackReporter) DefaultAndValidate() error {
 	return nil
 }
 
+// WebhookReporter represents the config for the webhook reporter. The URL can be overridden
+// on the job via the .reporter_config.webhook.url property.
+type WebhookReporter struct {
+	JobTypesToReport              []prowapi.ProwJobType `json:"job_types_to_report,omitempty"`
+	prowapi.WebhookReporterConfig `json:",inline"`
+}
+
+// WebhookReporterConfigs represents the config for the webhook reporter(s).
+// Use `org/repo`, `org` or `*` as key and a `WebhookReporter` struct as value.
+type WebhookReporterConfigs map[string]WebhookReporter
```

Comment:
Please add a section to `ProwJobDefaultEntry` rather than creating a new configuration matching system.
I know that you are following `SlackReporterConfigs` as an example, but that is a legacy example and we shouldn't mirror the tech debt.

Reply:
Thanks for the review. Can you please provide me with some guidance? Could you please point me to where a default entry is set for another reporter?